subsys/mpsl/fem/nrf2220/mpsl_fem_nrf2220.c

```diff
@@ -113,7 +113,10 @@ static int fem_nrf2220_configure(void)
 			},
 			.enable        = true,
 			.active_high   = MPSL_FEM_GPIO_POLARITY_GET(cs_gpios),
-			.gpiote_ch_id  = cs_gpiote_channel
+			.gpiote_ch_id  = cs_gpiote_channel,
+#if defined(NRF54L_SERIES)
+			.p_gpiote = cs_gpiote.p_reg,
```

Comment:
Can we have an explanation of the hardware connections for nRF2220 in the form of a comment section? Something like [this comments for simple gpio](https://projecttools.nordicsemi.no/bitbucket/projects/DRGN/repos/dragoon/browse/mpsl/libs/fem/src/simple_gpio/mpsl_fem_simple_gpio_dppi.c?at=3f1cacfd80a9db13bea3530327188262a560c410#17).

I have an analogic comment for nRF2240.

Reply:
@piotrkoziar  such doc is provided within the repo you pointed, just in other file.